### PR TITLE
support 3dlayout

### PR DESCRIPTION
**Pull request by ring630**

### CODE REVIEW

File: aedttest/aedt_test_runner.py [hunk 1]
```diff
@@ -802,6 +802,9 @@ def copy_proj(project_config: Dict[str, Any], dst: str) -> Union[str, List[str]]
 
     """
     src = project_config["path"]
+    src_aedb = src.replace(".aedt", ".aedb")
```
Comment (beliaev-maksim) [inline aedttest/aedt_test_runner.py:805]: I would use `pathlib`

**Review by beliaev-maksim — CHANGES_REQUESTED** [review]

File: aedttest/aedt_test_runner.py (hunk 1 above)
Comment (ring630) [reply aedttest/aedt_test_runner.py:805]: Done :D

**Comment by ring630** [comment]
@beliaev-maksim pre-commit is failing. It says configuration is not valid. What could be wrong here?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ jobs:
 
     - uses: actions/setup-python@v2
       with:
-        python-version: 3.7
+        python-version: "3.10"
```
Comment (beliaev-maksim) [inline .github/workflows/ci.yml:19]: -1 on this

you do not ensure support for 3.7+

**Review by beliaev-maksim — DISMISSED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (beliaev-maksim) [reply .github/workflows/ci.yml:19]: find a way to satisfy all :)

Comment (ring630) [reply .github/workflows/ci.yml]: @beliaev-maksim I found the root cause. rolled back and set isort to 5.11.5. but pre-commit is still failing. There is no python27 on git.

    RuntimeError: failed to find interpreter for Builtin discover of python_spec='python2.7'

Comment (ring630) [reply .github/workflows/ci.yml]: @beliaev-maksim removed python2.7 from pre-commit. Please inspire me if there is a better solution.

Comment (beliaev-maksim) [reply .github/workflows/ci.yml]: we might need to install it

but @boyang2022 can answer better

Comment (ring630) [reply .github/workflows/ci.yml]: @beliaev-maksim it is best to have both python 3.7 and 2.7 in pre-commit, but I have no idea how to setup two python environments. Can you help me on it?

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM

**Pull request merged**